GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -485,17 +528,42 @@ gdjs.HowlerSoundManager.prototype.unloadAudio = function (soundName, isMusic) {
 
   if (!container[soundFile]) return;
 
+  // Make sure any sound using the howl is deleted so that the howl can be garbage collected
+  // and no weird "zombies" using the unloaded howl can exist.
+  const howl = container[soundFile];
+  function removeFromConatiner(soundContainer) {
+    for (let i in soundContainer) {
+      if (soundContainer[i] && soundContainer[i]._howl === howl) {
+        soundContainer[i].stop();
```

Comment:
Should you call `unload` instead of stop?

Reply:
I cannot unload a `HowlerSound`, as it is basically sharing the Loaded Howl and the difference is only data like the seek, the played state etc. I shouldn't have to stop them, I just do to make sure they don't try to continue playing the unloaded Howl to prevent "uncontrollable sounds".

Reply:
Ok I see. Considering this ok then, though might not another check in case we go with a `destroy` method.